.travis.yml

```diff
@@ -4,6 +4,7 @@ language: c
 matrix:
     include:
         - os: linux
+          compiler: gcc
```

Comment:
We have to explicitly install `gcc` in docker build. This should not be needed.

Reply:
It's basically a no-op to make it clear to someone reading this config file that we want gcc for the Linux builds. It's to mirror the `compiler: clang` entry that isn't a no-op for the macOS environment. Yes, it's redundant but I think warranted in this case.